firmware/network/radio/include/radio.h

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @{
+ * @ingroup     network
+ * @file        radio.h
+ * @brief       this module content all functions of radio
+ * @author      xkevin190 <kevinvelasco193@gmail.com.com>
+ * @author      eduazocar <eduazocarv@gmail.com>
+ * @author      RocioRojas <rociorojas391@gmail.com>
+ *
+ */
+
+#ifndef RADIO_H
+#define RADIO_H
+
+#include "net/gnrc.h"
+#include "net/netdev.h"
+
+#ifndef MAX_TX_POWER
+#define MAX_TX_POWER (20)
+#endif
+
+#ifndef MIN_TX_POWER
+#define MIN_TX_POWER (-17)
+#endif
+
```

Comment:
why you have this defines here?  doesn't it is in the makefile?

Reply:
These params are in case that it's used an uknown driver